receiver/receiver_functions.py

```diff
@@ -0,0 +1,33 @@
+temperature_range = []
+state_of_charge = []
+    
```

Comment:
The name of this file - `receiver_functions` is not specific. It doesn't indicate what can be in here and what not. Also, the folder is already `receiver`, so each function doesn't need to have the prefix again.
How about `compute_stats.py`? If you name it like this, you realize that the `displayoutput` functionality doesn't fit in here. That will result in a new file called `output.py`, which is a good decomposition - the output format can evolve independent of the statistics-computation.